examples/interdomain/nsm_consul/README.md

```diff
@@ -65,7 +65,7 @@ kubectl --kubeconfig=$KUBECONFIG1 exec pod/dashboard-nsc -c cmd-nsc -- curl coun
 
 Port forward and check connectivity from NSM+Consul by yourself!
 ```bash
-kubectl --kubeconfig=$KUBECONFIG1 port-forward pod/dashboard-nsc 9002:9002 &
+kubectl --kubeconfig=$KUBECONFIG1 port-forward pod/dashboard-nsc 9002:9002 & sleep 60
```

Comment:
Why do we need a sleep here?

Reply:
If add 'sleep' here, than output will be ordered:

Reply:
It's mostly for visability in logs. 
If add 'sleep' here, output will be ordered:

> kubectl --kubeconfig=$KUBECONFIG2 -n ns-nsm-consul-vl3 port-forward dashboard 9002:9002 & sleep 60 TestRunBasicInterdomainSuite/TestNsm_consul_vl3=stdin
Forwarding from 127.0.0.1:9002 -> 9002
Forwarding from [::1]:9002 -> 9002 TestRunBasicInterdomainSuite/TestNsm_consul_vl3=stdout
result=$(curl --include --no-buffer --connect-timeout 20 -H "Connection: Upgrade" -H "Upgrade: websocket" -H "Host: 127.0.0.1:9002" -H "Origin: http://127.0.0.1:9002/" -H "Sec-WebSocket-Key: SGVsbG8sIHdvcmxkIQ==" -H "Sec-WebSocket-Version: 13" http://127.0.0.1:9002/socket.io/?EIO=3&transport=websocket) TestRunBasicInterdomainSuite/TestNsm_consul_vl3=stdin

If not, then something like this will be in the output:

> kubectl --kubeconfig=$KUBECONFIG1 port-forward pod/dashboard-nsc 9002:9002 & 
> kubectl --kubeconfig=$KUBECONFIG2 delete deploy counting
> deployment.apps "counting" deleted 
> kubectl --kubeconfig=$KUBECONFIG1 apply -f https://raw.githubusercontent.com/networkservicemesh/deployments-k8s/189e85b1526bdd18dcf509c8147341d3f73a7215/examples/interdomain/nsm_consul/server/counting_nsm.yaml 
> Forwarding from 127.0.0.1:9002 -> 9002
> Forwarding from [::1]:9002 -> 9002
> serviceaccount/counting created
>